code/modules/events/scrubber_overflow.dm

```diff
@@ -3,7 +3,9 @@
 	typepath = /datum/round_event/scrubber_overflow
 	weight = 10
 	max_occurrences = 3
-	min_players = 10
+	min_players = 10 MINUTES
+	category = EVENT_CATEGORY_ENGINEERING
```

Comment:
``///Events that cause breakages and malfunctions that could be fixed by engineers``
This is more like a Janitor one, maybe this (and scrubber clog) can be moved to a new Janitor one?